baggage/baggage_test.go

```diff
@@ -657,7 +660,10 @@ func TestMemberProperties(t *testing.T) {
 }
 
 func TestMemberValidation(t *testing.T) {
-	m := Member{}
+	m := Member{isEmpty: true}
+	assert.ErrorIs(t, m.validate(), errInvalidMember)
+
+	m.isEmpty = false
 	assert.ErrorIs(t, m.validate(), errInvalidKey)
```

Comment:
You removed a Valid and important test.  Specifically that `Member{}`, a thing that users can do, should report they are invalid.

Reply:
See my other comment.